crates/wire/src/p2p/v1.rs

```diff
@@ -0,0 +1,6 @@
+pub mod proto {
+    include!(concat!(env!("OUT_DIR"), "/strata.bitvm2.p2p.v1.rs"));
```

Comment:
Do we need this barrage of macros?

Reply:
Unfortunately, yes, this is an official usage of `prost-build`

Reply:
Ok, TIL.